test/e2e/operator.go

```diff
@@ -21,7 +21,7 @@ import (
 
 const (
 	localHelmChart            = "helm-chart/dash0-operator"
-	operatorHelmReleaseName   = "e2e-tests-operator-helm-release"
+	operatorHelmReleaseName   = "e2e-tests-operator-hr"
```

Comment:
combining the release name with the target-allocator deployment name resulted in a string >=64 chars